gpu/kinfu/tools/kinfu_app.cpp

```diff
@@ -128,18 +128,17 @@ namespace pcl
         }
               
         /** \brief Obtain the actual color for the input dataset as vtk scalars.
-          * \param[out] scalars the output scalars containing the color for the dataset
-          * \return true if the operation was successful (the handler is capable and 
-          * the input cloud was given as a valid pointer), false otherwise
+          * \return Array of scalars if the operation was successful (the handler is capable and 
+          * the input cloud was given as a valid pointer), nullptr otherwise
           */
-        bool
-        getColor (vtkSmartPointer<vtkDataArray> &scalars) const override
+        vtkSmartPointer<vtkDataArray>
+        getColor () const override
```

Comment:
What about the call-site? It needs to be modified too right?

Reply:
No, it is in PCLVisualizer, which has the deprecated method yet to be removed in #4341 .